scripts/utils/helpers.ts

```diff
@@ -344,6 +348,122 @@ const genesisDeriveFutureAddress = async function (
   return contractDeterministic.address;
 };
 
+const getGasLimit = async function (
+  hre: LeanHardhatRuntimeEnvironment,
+  from: string | SignerWithAddress | SuperColdStorageSigner,
+  to: string,
+  data: Promise<UnsignedTransaction> | BytesLike = '',
+  value: BigNumberish = 0
+): Promise<BigNumber> {
+  if (typeof from !== 'string') {
+    from = (from as SignerWithAddress).address;
+  }
+  if (!isBytesLike(data)) {
```

Comment:
should this check for false for clarity?

Reply:
Good idea

Reply:
Fixed